bigbluebutton-html5/imports/ui/components/meeting-ended/component.jsx

```diff
@@ -167,7 +167,11 @@ class MeetingEnded extends React.PureComponent {
       <div className={styles.parent}>
         <div className={styles.modal}>
           <div className={styles.content}>
-            <h1 className={styles.title}>{intl.formatMessage(intlMessage[code])}</h1>
+            <h1 className={styles.title}>
+              {
+                intl.formatMessage(intlMessage[code] || intlMessage[430])
```

Comment:
You should log when the message can't be found. With this we will continue to silently fallback and have no way to know about it to improve the code.